prime-router/docs/design/design/RS-Extention-Registry.yml

```diff
@@ -0,0 +1,5930 @@
+        }
+      Extension:
+        - id: software-vendor-org
+          url: "https://reportstream.cdc.gov/fhir/StructureDefinition/software-vendor-org"
+          valueType: Reference
+          hl7Use: SFT.1
+        - id: software-binary-id
+          url: "https://reportstream.cdc.gov/fhir/StructureDefinition/software-binary-id"
+          valueType: String
+          hl7Use: SFT.4
+        - id: source-software-description
+          url: "https://reportstream.cdc.gov/fhir/StructureDefinition/source-software-description"
+          valueType: String
+          hl7Use: SFT.5
+        - id: software-install-date
+          url: "https://reportstream.cdc.gov/fhir/StructureDefinition/software-install-date"
+          valueType: String
+          hl7Use: SFT.6
+    HD:
+      Example:
+        {
+          "fullUrl": "MessageHeader/827ccb0e-ea8a-306c-8c34-a16891f84e7b",
+          "resource": {
+            "resourceType": "MessageHeader",
+            "id": "827ccb0e-ea8a-306c-8c34-a16891f84e7b",
+            "extension": [
+              {
+                "url": "https://reportstream.cdc.gov/fhir/StructureDefinition/encoding-characters",
+                "valueString": "^~\\&#"
+              },
+              {
+                "url": "https://reportstream.cdc.gov/fhir/StructureDefinition/character-set",
+                "valueString": "UNICODE UTF-8"
+              }
+            ],
+            "source": {
+              "extension": [
+                {
+                  "url": "https://reportstream.cdc.gov/fhir/StructureDefinition/namespace-id",
+                  "valueString": "Sending App"
+                },
+                {
+                  "url": "https://reportstream.cdc.gov/fhir/StructureDefinition/universal-id",
+                  "valueString": "Sending App Universal"
+                },
+                {
+                  "url": "https://reportstream.cdc.gov/fhir/StructureDefinition/universal-id-type",
+                  "valueString": "ISO"
+                },
+                {
+                  "url": "https://reportstream.cdc.gov/fhir/StructureDefinition/hl7v2Field",
+                  "valueString": "MSH.3"
+                }
+              ]
+            }
+          }
+        }
+      Extension:
+        - id: namespace-id
+          url: "https://reportstream.cdc.gov/fhir/StructureDefinition/namespace-id"
+          valueType: String
+          hl7Use: HD.1
+        - id: universal-id
+          url: "https://reportstream.cdc.gov/fhir/StructureDefinition/universal-id"
+          valueType: String
+          hl7Use: HD.2
+        - id: universal-id-type
+          url: "https://reportstream.cdc.gov/fhir/StructureDefinition/universal-id-type"
+          valueType: String
+          hl7Use: HD.3
+        - id: hl7v2Field
+          url: "https://reportstream.cdc.gov/fhir/StructureDefinition/hl7v2Field"
+          valueType: String
+          hl7Use: NA
+          constant: "MSH.3", "MSH.24"
+
+- Resource: Any
```

Comment:
Should this be `Attachment`?

Reply:
I didn't include this as Resource since we don't use it as a resource in our mapping. We only ever use `valueAttachment` in extensions